src/main/java/de/tum/in/www1/artemis/web/rest/CourseResource.java

```diff
@@ -106,11 +108,16 @@ public class CourseResource {
 
     private final CourseScoreCalculationService courseScoreCalculationService;
 
+    private final ConversationService conversationService;
+
+    private ChannelRepository channelRepository;
```

Comment:
Is there a reason why this is not final?

Reply:
Actually no, `final` will be added.